Week 3/shopee-reduxtoolkit-typscript/src/components/Product.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+
+import { Link, useNavigate } from 'react-router-dom';
+import { useAppDispatch } from '../store/store';
+import { addToCart } from "../store/productSlice";
+import { faEye } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+
+const Product = ({ id, title, price, desc, image }: { id: String | undefined, title: String | undefined, price: Number, desc: String | undefined, image: String }) => {
```

Comment:
@phamhuutinh0605 : định nghĩa type IProductProps không để trong function vậy sẽ khó đọc code
```suggestion
const Product = ({ id, title, price, desc, image }: IProductProps) => {
```